src/components/app/list/list.css

```diff
@@ -298,25 +299,19 @@
 }
 
 .sort-up {
-    background-image: url('../../../assets/icons/misc/sort-up.svg');
+    background-image: url('../../../assets/icons/ic-sort-arrow-down.svg');
```

Comment:
Check if sort-up & down is used anywhere & confirm changes accordingly. Also, try to extract common out.